assets/js/controllers/buySell.controller.js

```diff
@@ -59,16 +58,10 @@ function BuySellCtrl ($rootScope, $scope, $state, Alerts, Wallet, currency, buyS
       $scope.sellTransaction.currency = buySell.getCurrency();
     }, true);
 
-    $scope.$watch('transaction.currency', (newVal, oldVal) => {
-      let curr = $scope.transaction.currency || null;
-      $scope.currencySymbol = currency.conversions[curr.code];
-      if (newVal !== oldVal) $scope.getMaxMin($scope.limits, $scope.transaction);
-    });
-
     $scope.$watch('sellTransaction.currency', (newVal, oldVal) => {
       let curr = $scope.sellTransaction.currency || null;
       $scope.sellCurrencySymbol = currency.conversions[curr.code];
-      if (newVal !== oldVal) $scope.getMaxMin($scope.sellLimits, $scope.sellTransaction);
+      // if (newVal !== oldVal) $scope.getMaxMin($scope.sellLimits, $scope.sellTransaction);
```

Comment:
@prwelber we'll need to figure out how to handle max min amounts for sell. Maybe you can do something similar to what I'm doing for now with buy.